dbt/include/hive/macros/adapters.sql

```diff
@@ -101,14 +101,28 @@
 {% endmacro %}
 
 
-{% macro properties_clause(properties) %}
+{% macro properties_clause() %}
+  {%- set properties = config.get('properties') -%}
+  {%- set table_type = config.get('table_type') -%}
+  {%- set format_version = config.get('format_version') -%}
```

Comment:
customer can also provide the format_version as https://github.com/cloudera/dbt-impala/pull/174/files#diff-8918058eee4cfaebfe8d4ffe174b6acc56900227cc2790320fd5690da34e4391R65

Reply:
Made changes accordingly